### PR TITLE
osd: use existing osd_required variable for messenger policy

### PR DESCRIPTION
**Pull request by Yan-waller**
Signed-off-by: Yan Jun <yan.jun8@zte.com.cn>

### CODE REVIEW

File: src/ceph_osd.cc [hunk 1]
```diff
@@ -529,33 +529,30 @@ int main(int argc, const char **argv)
...
+  ms_cluster->set_policy(entity_name_t::TYPE_MON,
+                         Messenger::Policy::lossy_client(0));
   ms_cluster->set_policy(entity_name_t::TYPE_OSD,
 			 Messenger::Policy::lossless_peer(osd_required));
   ms_cluster->set_policy(entity_name_t::TYPE_CLIENT,
 			 Messenger::Policy::stateless_server(0));
 
   ms_hb_front_client->set_policy(entity_name_t::TYPE_OSD,
-			  Messenger::Policy::lossy_client(0));
+                                 Messenger::Policy::lossy_client(0));
```
Comment (tchaikov) [inline src/ceph_osd.cc]: i'd suggest avoid s/\tab/        / changes if possible.

Comment (xiexingguo) [reply src/ceph_osd.cc]: yeah, this is a pretty old file. And this kind of formatting nits will make it more difficult to backport:(

Comment (Yan-waller) [reply src/ceph_osd.cc]: OK, fixed it.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2018-02-05_11:48:09-rados-wip-kefu-testing-2018-02-05-1650-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-02-06_06:20:39-rados-wip-kefu-testing-2018-02-05-1650-distro-basic-mira/

**Pull request merged**